scripts/migration/migrate_registered_meta.py

```diff
@@ -0,0 +1,78 @@
+"""
```

Comment:
A migration script! (my, how very prompt)

Please add a note about this to the ticket description / deployment notes, as you feel would be most appropriate.